File: ws/viteri/u2/WS24Zo0/Zoo/src/main/java/com/mycompany/zoo/view/Zoo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.zoo.view;

import java.util.ArrayList;
import java.util.Date;
import model.Animal;
import model.Cage;
import model.Food;
import model.Lion;
import model.Monkey;

/**
 *
 * @author Alexis Viteri DCO-ESPE
 */
public class Zoo {
    public static void main(String[] args) {
        System.out.println("Alexis's Zoo");
        Food food=new Food(3, "fish");
        Cage cageFelines= new Cage(1,"felinesCage");
        Cage cageMonkeys= new Cage(2,"monkeysCage");
        Cage cageBirds= new Cage(3,"birdsCage");
        Cage pool= new Cage(4,"SharkCage");
       // ArrayList;
        Animal animal = null;
       // lion.register();
       Lion lion=new Lion(1, "Alex el Leon", new Date(2024,10,10),'m', 0,cageFelines , 0, true);
       lion.register(cageFelines);
       //animals.add(Lion);
       lion.feed(5);
        System.out.println("animal(lion)--->"+animal);
       animal=new Monkey(2, "Pancho" ,new Date(2024,10,10),'m', 2, cageMonkeys, 0, true);
       animal.register(pool);
       animal.feed(4);
        System.out.println("animal(Monkey)---->"+ animal);
        
       
    }
}

File: hw/viteri/u2/HW18ZooInPython/Zoo/view/Zoo.py
from datetime import date
from model.Food import Food
from model.Cage import Cage
from model.Lion import Lion
from model.Monkey import Monkey

def main():
    print("Alexis's Zoo")

    food = Food(3, "fish")
    cage_felines = Cage(1, "felinesCage")
    cage_monkeys = Cage(2, "monkeysCage")
    cage_birds = Cage(3, "birdsCage")
    pool = Cage(4, "SharkCage")

    lion = Lion(1, "Alex el Leon", date(2024, 10, 10), 'm', 0, cage_felines, 0, True)
    lion.register(cage_felines)
    lion.feed(5)
    print(f"animal(lion)--->{lion}")

    monkey = Monkey(4, "Pancho" , 10-20-2024, "m" , 2, 23, True)
    monkey.register(pool)
    monkey.feed(4)
    print(f"animal(Monkey)---->{monkey}")

if __name__ == "__main__":
    main()